apps/picsa-tools/resources-tool/src/app/schemas/base/schema_v0.ts

```diff
@@ -25,6 +25,8 @@ export interface IResourceBase_v0 {
   priority?: number;
   /** additional metadata populated for specific resource types */
   meta?: any;
+  /** Languages the resource is available in */
+  languages?: string[];
```

Comment:
In order to accommodate the list of languages a resource video is available in, I added a `languages` array to the schema. Any comments on this? @chrismclarke